deepfence_server/pkg/integration/s3/s3.go

```diff
@@ -63,14 +63,18 @@ func (s S3) SendNotification(ctx context.Context, message string, extras map[str
 		fmt.Println("Failed to marshal JSON data", err)
 		return err
 	}
-
+	buf := &bytes.Buffer{}
```

Comment:
Can we have a buffer that is always present ? We create it once, use it to compress and then clear it out ?